packages/core/src/core/createEvent.js

```diff
@@ -126,6 +129,14 @@ export default () => {
         event.mergeData(userData);
       }
 
+      if (content?.xdm?.timestamp) {
```

Comment:
Won't there always be a timestamp inserted by the context component (https://vscode.dev/github/adobe/alloy/blob/impr/noStampOnlyQueue/packages/core/src/components/Context/index.js#L46)?